### ISSUE TITLE
When resizing the text on 'Products' page, categories name's sizes are not changing.

### ISSUE THREAD

**Issue by Shalini-git25** [DFE-Digital/find-information-products-services-frontend#25]
**Describe the bug**
Using 'Resize text' tool on Products page to resize all text to meet accessibility standards. All text are resizing properly except for Categories name -Phase, Channel, Group and Type text sizes are not changing.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Products' page.
2. Select the 'resize text' tool.
3. Categories name text - Phase, Channel, Group and Type are not resizing as expected.
4. It's not meeting accessibility standards.

**Expected behavior**
All the text including Categories name in Products page should be resized properly to meet accessibility standards.

**Screenshots**

<img width="2218" height="1326" alt="Image" src="https://github.com/user-attachments/assets/55224382-db48-4f3c-8d72-b893d54c66a0" />
